test/API/H5_api_dataset_test.c

```diff
@@ -11531,19 +11427,6 @@ test_dataset_set_extent_invalid_params(void)
         goto error;
     }
 
-    /** for DAOS VOL, this test is problematic since auto chunking can be selected, so skip for now */
-    if (H5VLget_connector_name(file_id, vol_name, 5) < 0) {
```

Comment:
I'm guessing this was removed because this test can now be skipped during DAOS testing at runtime?

Reply:
Yes, I believe my thinking at the time was that it's not really necessary to have VOL-specific hacks in the code (I don't like these anyway) since this PR allows us to have more fine-grained control over skipping particular tests.